data.js

```diff
@@ -7,6 +7,14 @@
 // also be just the raw JS if it's small enough, an array of source files or a ZIP file.
 // See examples below.
 module.exports = [
+  {
+    name: "defer.js",
+    github: "shinsenter/defer.js",
+    tags:  ["deferjs", "defer loading", "lazy loading js", "lazyloader", "lazyload", "native", "scroll reveal", "scroll monitor", "intersection observer"],
+    description: "🥇 A super small, super efficient library that helps you lazy load everything like images, video, audio, iframes as well as stylesheets, and JavaScript.",
+    url: "https://github.com/shinsenter/defer.js",
+    source: "https://raw.githubusercontent.com/shinsenter/defer.js/master/dist/defer.min.js"
```

Comment:
Please change this to provide a human-readable source file.

From the microjs readme:

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified